packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -385,6 +386,11 @@ const registerCommands = (
     renameLightningComponent
   );
 
+  const getCoreLoggerServiceCmd = vscode.commands.registerCommand(
+    'sf.rename.lightning.component',
```

Comment:
shouldn't this be `"sf.vscode.core.logger.get.instance"`?

Reply:
YES